tests/profiling/exporter/test_http.py

```diff
@@ -466,3 +466,34 @@ def test_get_tags_legacy(monkeypatch):
     assert tags["mytag"] == "val2"
     assert tags["ddtag"] == "hi"
     assert tags["ddptag"] == "lo"
+
+
+def test_gitmetadata_env(monkeypatch):
```

Comment:
can we have an example that only has the values set in `DD_TAGS`, since we are removing them from the tags.